pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/TabularParticipantParser.java

```diff
@@ -241,7 +241,11 @@ private void addModuleDataToParticipantMap(ModuleExportConfig moduleConfig, Map<
                 if (filterConfig.getChildConfigs() != null) {
                     // if this is a composite question, we need a separate column for each child question
                     for (int childIndex = 0; childIndex < filterConfig.getChildConfigs().size(); childIndex++) {
-                        addSingleResponseToMap(Collections.singletonList(responseValues.get(childIndex)),
+                        List<String> childResponse = new ArrayList<>();
+                        if (childIndex < responseValues.size()) {
+                            childResponse = Collections.singletonList(responseValues.get(childIndex));
+                        }
+                        addSingleResponseToMap(childResponse,
```

Comment:
(Just a thought) I am wondering, if the middle childConfigs have an answer, won't this change the order of how the response is added to the export?